.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,25 @@
+## Description
+
 <!--
-    If this is a security-related patch stop immediately!
-    See http://docs.django-cms.org/en/latest/contributing/development-policies.html
+If this is a security issue stop right here and follow our documentation:
```

Comment:
``... stop immediately and follow the instructions at http://docs.django-cms.org/en/latest/contributing/development-policies.html#reporting-security-issues.``